3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/sqlquery/OutputSpec.scala

```diff
@@ -42,6 +42,6 @@ class OutputSpec extends Specification with ValidationMatchers { def is =
 
   def e4 = {
     val date = new Date(1465558727000L)
-    JsonOutput.getValue(date, "java.sql.Date") must beEqualTo(JString("2016-06-10T11:38:47.000Z"))
+    JsonOutput.getValue(date, "java.sql.Date") must beEqualTo(JString("2016-06-10T13:38:47.000+02:00"))
```

Comment:
Same here, given that the json4s version didn't change I don't why the default date formats would have changed.